File: solutions/0167-two-sum-ii---input-array-is-sorted/solution.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i, n = 0, len(numbers)-1

        while(i != n):
            check = numbers[i]+numbers[n]
            if check == target:
                return [i+1,n+1]
                return res
            elif check > target:
                n -= 1
            else:
                i += 1
        

File: solutions/0167-two-sum-ii---input-array-is-sorted/solution.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int length = numbers.length;
        int l = 0;
        int r = length - 1;
        int sum = 0;
        while(l < r)
        {
            sum = numbers[l] + numbers[r];
            if(sum == target)
                return new int[] {l+1, r+1};
            else if(sum > target)
                r--;
            else
                l++;
        }
        return new int[] {0, 0};
    }
}
